packages/cxl-ui/src/components/cxl-tabs-slider.js

```diff
@@ -50,6 +57,46 @@ export class CXLTabsSliderElement extends Tabs {
     this.setAttribute(themeAttr, themes.join(' '));
   }
 
+  /** Redefining Vaadin-Tabs method to fix bug in scroll position end detection */
+  _updateOverflow() {
+    this.centeredContainerWidthCheck();
+
+    const scrollPosition = this._vertical
+      ? this._scrollerElement.scrollTop
+      : this.__getNormalizedScrollLeft(this._scrollerElement);
+    const scrollSize = this._vertical
+      ? this._scrollerElement.scrollHeight
+      : this._scrollerElement.scrollWidth;
+    const offset = this._vertical
+      ? this._scrollerElement.offsetHeight
+      : this._scrollerElement.offsetWidth;
+
+    // Note that we are not comparing floored scrollPosition to be greater than zero here, which would make a natural
+    // sense, but to be greater than one intentionally. There is a known bug in Chromium browsers on Linux/Mac
+    // (https://bugs.chromium.org/p/chromium/issues/detail?id=1123301), which returns invalid value of scrollLeft when
+    // text direction is RTL. The value is off by one pixel in that case. Comparing scrollPosition to be greater than
```

Comment:
> which returns invalid value of scrollLeft when text direction is RTL

We have LTR on the site only. Does it really affect us? Or it affect users whose system uses RTL?

Reply:
That's the same code found in vaadin-tabs. I needed to redefine the entire function in cxl-tabs-slider to fix a problem that only affects our component. To avoid unexpected behavior, I kept the same exact code for the other parts of the function. Also, yes, it might affect users that prefer RTL, depending if they enforce it in the browser, so I would keep it as is.